docs/4.1/admin-guide.md

```diff
@@ -1991,6 +1991,9 @@ belong to:
 $ tctl users add joe --k8s-groups="system:masters"
 ```
 
+!!! tip "NOTE":
+    In order to execute the above command, a Teleport role in RBAC needs to be created as `k8s-group`. An example RBAC role can be found in the [Kubernetes SSH](https://gravitational.com/teleport/docs/kubernetes_ssh/#impersonation) section.
```

Comment:
For internal links we are using mkdocs functionally, so we can run automated tests on links.

This should be 
`kubernetes_ssh.md#impersonation`

Reply:
That's right, just committed this update